events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -201,8 +213,9 @@ <h4>Choose Question Details (Optional) <br>& Click 'New Question'</h4>
                         triviaData = JSON.parse(event);
                     } catch (e){
                         console.log(e, "parsing error");
+                        return;
                     }
-                    if (!triviaData.application || triviaData.application !== "trivia") {
+                    if (!event || !triviaData || !triviaData.application || triviaData.application !== "trivia") {
```

Comment:
the code couldn't even get here if `event` was falsey, right?

Reply:
I didn't think so, but strangely enough it did!  Super weird.

Reply:
oh ok, i see the logic now! np